ci/provisioning/post_provision_config.sh

```diff
@@ -24,6 +24,13 @@ DSL_REPO_var="DAOS_STACK_${DISTRO}_LOCAL_REPO"
 DSG_REPO_var="DAOS_STACK_${DISTRO}_GROUP_REPO"
 DSA_REPO_var="DAOS_STACK_${DISTRO}_APPSTREAM_REPO"
 
+: "${CI_RPM_TEST_VERSION:=}"
+: "${DAOS_RPM_TEST_VERSION:=$CI_RPM_TEST_VERSION}"
+if [ -z "$DAOS_RPM_TEST_VERSION" ]; then
+    DAOS_RPM_TEST_VERSION="$(echo "$COMMIT_MESSAGE" |
+      sed -ne '/^RPM-test-version: */s/^[^:]*: *//Ip')"
+fi
+
```

Comment:
I'm failing to understand why this should be re-implemented here rather than simply putting this logic into the pre-existing `rpm_test_version()` function.  By doing this we now have disjoint methods of getting the result of a parameter value vs. a commit pragma evaluation, as we now have this one method here and have an alternate method in the `pr_repos()` function.

`pr_repos()` and `rpm_test_version()` were meant to be models to follow for further param vs. pragma evaluations.

Reply:
rpm_test_version() can not use the CI_RPM_TEST_VERSION environment variable in the child CI node scripts, and the only thing it was being used for is as a binary flag to enable the DAOS Artifactory repo.

So it is just simpler to pass an environment variable to be used as the flag, and that means that there is only one place that rpm_test_verison() would have been called, meaning it was not longer a common function.

It looks like pr_repos is not being called and would not work in the child CI node for the same reason is that the CI_PR_REPOS is not set on any of the child nodes.

All handling of pragma vs build parameters must either happen in the build agent node, or some way of passing all the build parameters to the CI nodes has to be created.

An alternate implementation would be to create a file with the current known pragma and build parameters that are meaningful to CI tests and copy it to each CI node before running the script.  Then a CI node function could make the evaluation.